picard/acoustid/manager.py

```diff
@@ -76,38 +80,39 @@ def submit(self):
             return
         log.debug("AcoustID: submitting ...")
         self.tagger.window.set_statusbar_message(
-            N_('Submitting AcoustIDs ...'),
-            echo=None
+                N_('Submitting AcoustIDs ...'),
+                echo=None
         )
         self.tagger.acoustid_api.submit_acoustid_fingerprints(fingerprints,
-            partial(self.__fingerprint_submission_finished, fingerprints))
+                                                              partial(self.__fingerprint_submission_finished,
```

Comment:
No, it moves things too far on right, you'll end with wrapping a lot etc...
I know we don't enforce line width, but still, better have shorter lines.

Extreme example:

```python
somefunc(someverylongshitkjsdljfksldkfjskdjflkjdskfjlsdkjfkjsdkfksdjkfjsjjfsdk(ddddddddd,
                                                                               xxxxxxxxxx,
                                                                               yyyyyyyyyy),
                                                                               func(you_are_doomed,
                                                                                       messy))
```

```python
somefunc(
    someverylongshitkjsdljfksldkfjskdjflkjdskfjlsdkjfkjsdkfksdjkfjsjjfsdk(
        ddddddddd,
        xxxxxxxxxx,
        yyyyyyyyyy
    ),
    func(
        you_are_doomed,
        messy
    )
)